### PR TITLE
Model Submission for Coordinated Robotics Pam

### PR DESCRIPTION
**Pull request by acschang**
Continuation of #829 

**Adjustments:**
- Added RViz visualization

### CODE REVIEW

File: submitted_models/coro_pam_sensor_config_1/specifications.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+$540 - Frame / motors / props / batteries / controllers
+$500 -  Computer - GB-BRi5-8250
+$300 - LIDAR-Lite V3HP x 2
+$220 - lights / connectors /switches / DC-DC / misc
+
+
+## Sensors
+Pam Sensor Config 1 uses:
+* 3 x D435 modeled by rgbd_camera plugin
+* 1 x Kakute F7 - modeled by air_pressure plugin, imu_sensor and magnetometer plugin
```
Comment (jennuine) [inline submitted_models/coro_pam_sensor_config_1/specifications.md:31]: Looking at the Kakute 57 specs, it does not have an air_pressure sensor

Comment (acschang) [reply submitted_models/coro_pam_sensor_config_1/specifications.md:31]: On http://www.holybro.com/product/kakute-f7-v1-5/ there's a bullet that reads as follows:
> Ready for autonomous flight: Integrated BMP280 barometer and SCL/SDA pads for use with external GPS/magnetometer units.
> Specifications: * Barometer: BMP280
Regardless, barometer and magnetometer are standard on all UAVs in the SubT repository regardless of whether their real-world counterparts contain these sensors.

File: submitted_models/coro_pam_sensor_config_1/model.sdf [hunk 2]
```diff
@@ -0,0 +1,1231 @@
...
+        </geometry>
+      </visual>
+
+	    
+      <sensor name="rgbd_front" type="rgbd_camera">
+        <pose>0.105 0 -0.115 0 0 0</pose>
+        <always_on>1</always_on>
+        <update_rate>30</update_rate>
+        <camera name="rgbd_front">
+          <horizontal_fov>1.50098</horizontal_fov>
```
[the remark this answers is not in the document]
Comment (acschang) [reply submitted_models/coro_pam_sensor_config_1/model.sdf:769]: The current standard for RGBD sensors is to limit parameters to the characteristics of the depth camera.

File: submitted_models/coro_pam_sensor_config_1/model.sdf [hunk 3]
```diff
@@ -0,0 +1,1231 @@
...
+        <geometry>
+          <box>
+            <size>0.013 0.108 0.025</size>
+          </box>
+        </geometry>
+        <material>
+          <ambient>240 240 240 1</ambient>
+          <diffuse>240 240 240 1</diffuse>
+        </material>
+      </visual>
```
[the remark this answers is not in the document]
Comment (acschang) [reply submitted_models/coro_pam_sensor_config_1/model.sdf]: [Resolved](https://github.com/osrf/subt/pull/854/commits/5cb9198e4953842a9277f2426278fe4c95d0c178)

**Review by nkoenig — APPROVED** [review]

**Pull request merged**